django_filters/filters.py

```diff
@@ -737,12 +737,13 @@ def normalize_fields(cls, fields):
         ])
 
     def build_choices(self, fields, labels):
+        fields = {param: field for field, param in fields.items()}
         ascending = [
             (param, labels.get(field, _(pretty_name(param))))
-            for field, param in fields.items()
+            for param, field in fields.items()
         ]
         descending = [
-            ('-%s' % param, labels.get('-%s' % param, self.descending_fmt % label))
+            ('-%s' % param, labels.get('-%s' % fields[param], self.descending_fmt % label))
```

Comment:
I think the better approach here is to just call `pretty_name` again, instead of inverting the dict again. Something like...
```suggestion
            ('-%s' % param, labels.get('-%s' % field, self.descending_fmt % _(pretty_name(param))))
```

need to check if that's correct though..